setup.py

```diff
@@ -27,9 +27,9 @@
     license='BSD 3-clause "New" or "Revised" License',
     packages=['cromwell_tools'],
     install_requires=[
-        'requests==2.18.4',
-        'six==1.11.0',
-        'oauth2client==4.1.2',
+        'requests>=2.18.4',
```

Comment:
Thanks for making these requirements less restrictive! How about making `tenacity` and `setuptools_scm` less restrictive as well?

Reply:
Yea, pip v10 is now announcing all the crazy incompatibilities. :(

Reply:
Sure, I can do it for both of them!

Reply:
@ambrosejcarr Would you mind sharing your experience about pio v10 here? I heard a lot of people talking about it, but never run into any issue yet(and I don't want to)